src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DisposeObjectsBeforeLosingScopeTests.cs

```diff
@@ -11774,5 +11774,88 @@ class Test
     }
 }");
         }
+
+        [Fact, WorkItem(3085, "https://github.com/dotnet/roslyn-analyzers/issues/3085")]
+        public async Task LocalInvocationOfAnExcludedType_NoDiagnostic()
```

Comment:
Do we want more tests for the syntax of `excluded_symbol_names`?

Reply:
I think one test is good enough here.